SQL/Archive/25.0/2022-03-03-AddHEDTags.sql

```diff
@@ -44,6 +44,7 @@ INSERT INTO physiological_event_file (PhysiologicalFileID, FilePath, FileType)
 UPDATE physiological_task_event te
     SET EventFileID=(SELECT EventFileID FROM physiological_event_file WHERE PhysiologicalFileID=te.PhysiologicalFileID)
 ;
+SET FOREIGN_KEY_CHECKS= 1;
```

Comment:
I don't get why this is flagged as "IMPORTANT" in the description. It's a session variable, it'll be re-enabled after the patch is sourced even with this missing.

Reply:
the patch is the entire release patch because its concatenated into it

Reply:
I think it is fine, the FOREIGN_KEY_CHECKS is disabled for the same set of statements on both patches.